src/fastly/include-util.js

```diff
@@ -13,6 +13,14 @@
 const fs = require('fs-extra');
 const path = require('path');
 
+function regex(content, filePath) {
+  return content.replace(/\{"regex:block.rgx"\}/g, () => {
+    const name = path.resolve(filePath);
```

Comment:
`filePath` should probably be named `relPath` and `name` should be named `filePath`.

does `filePath` really need to be resolved?